app/Dto/Requests/Admin/Doctor/ShowDoctorsRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace backend.Dto.Requests.Admin.Doctor
+{
+    public class ShowDoctorsRequest
```

Comment:
Wiem, że tego nie zrobiłem przy okazji robienia paginacji ale dopiero potem na to wpadłem, że można zrobić  request do paginacji z tym polem i te nasze requesty po nim dziedziczyć lub tak jak w tym przypadku jego bezpośrednio używać jeśli nie ma innego filtrowania

Reply:
zrobione